e2e/integration/ssitests.js

```diff
@@ -12,45 +12,61 @@ context( 'Test for SSI', () => {
     } );
 
     it( 'Test for Notifications section', () => {
-        cy.get( '.op-nav-wrapper' ).within( () => {
-            cy.contains( 'Notifications' ).click();
-        } );
-        cy.get( '#op-menu-drawer' ).should( 'be.visible' ).within( () => {
-            cy.get( '.op-menu-drawer__title' ).should( 'contain.text', 'All Notifications ' );
+        cy.get( 'slot[name="opc-nav-btn"] > button:first-child' ).click({force:true}).then( () => {
+            cy.wait( 2000 )
+            cy.get( '.opc-notification-drawer__header-title',{force:true} ).invoke( 'text' ).then( ( text ) => {
+                expect( text.trim()).equal('Notifications' )
+                })
+
+
         } );
-    } );
+        cy.get(".opc-notification-drawer__header > button").click({force:true})
```

Comment:
@mudit94 I think this would be valid only if there are no id's

Reply:
@hybridx , this is css selector way of locating element